hw/ip/otbn/data/insns.yml

```diff
@@ -1724,8 +1726,8 @@ insns:
       - If `grs1_inc` is set, the value in `grs1` is incremented by the value WLEN/8 (one word).
       - If `grd_inc` is set, the value in `grd` is incremented by the value 1.
 
-      Only aligned (to WLEN sized regions) accesses are supported.
-      Any unaligned access will result in an error.
+      The memory address must be aligned to WLEN bytes.
+      Any unaligned or out-of-bounds address will result in an error (with error code `ErrCodeBadDataAddr`).
 
       TODO: how to handle overflows?
```

Comment:
I think this answers that TODO?

Reply:
I thought the TODO might be about overflows for the address update, so left it in. Ah, but I guess that can't happen because we only allow "++" (not "--") and we're unlikely to have valid data with an address of `2^32 - 4` :-)

About the clarification about "out of bounds", I meant "above the top of memory". I'll change the text to that now (I was avoiding needing to say "or negative", but I guess addresses end up unsigned anyway, so it doesn't need saying). I don't want to write down the size of memory here: that's defined somewhere else. Does that sound ok?

Reply:
Ah yes you're right, maybe best left in for now. The answer may be it just doesn't happen because you'd need to be close to the top of address space at which point you hit an out of bounds error before you worry about increments.

Fine not to write down the size of memory just make up a constant `DMEM_SIZE` or similar for the pseudo-code